### DISCUSSION TITLE
Data from Google Sheet [Oct 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4157] [Discord Threads]
> 🙋‍♂️ **Artificial Creative**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### You can download as Excel/CSV & upload that into a table. No integration directly with Google sheets for now  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1028477276129607680)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
